components/Blog/Card/Title.tsx

```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image';
+import Link, { LinkProps } from 'next/link';
+
+interface Props {
+   link: LinkProps;
+   children: React.ReactNode;
+}
+
+const Title = ({ children, link }: Props) => {
+   return (
+      <div className="flex items-center justify-between">
+         <h3 className="text-2xl font-semibold text-gray-900">{children}</h3>
+         <Link {...link}>
+            <Image
```

Comment:
You should not use svgs as Image. Svg is not an image type. It is vector. Please use svg component library which already installed to projrct named as react-feather. Or you can add new svg icons and use with svgr.